### ISSUE TITLE
trim front option to allow more than 30bp

### ISSUE THREAD

**Issue by cirodri1** [OpenGene/fastp#300]
Hello! I am very thankful for this tool as I use it all the time!

However, I am running into some issues as I need to cut the first 50 bp of my reads but the program does not allow me 

ERROR: trim_front1 (--trim_front1) should be 0 ~ 30, suggest 0 ~ 4

I see that this issue was brought up before [here](https://github.com/OpenGene/fastp/issues/121?_pjax=%23js-repo-pjax-container) but the issue was closed despite having some demand for this implementation.

Is there a way to allow this in the program? Or what alternatives do you recommend?

**Comment by shinthor** [comment]
I agree this should be changed to a warning instead of an error, there are plenty of cases where you may want to trim more than the ranges allowed here.
